dts/x86/intel_quark_d2000.dtsi

```diff
@@ -0,0 +1,57 @@
+#include "skeleton.dtsi"
+#include "mem.h"
+
+/ {
+	cpus {
+		cpu@0 {
+			compatible = "intel,quark";
```

Comment:
quark_d2000 and quark_se_c1000 have same cpu. But quark_se_c1000 also have sensor subsystem which is arc cpu which not present in quark_d2000